GitUI/Translation/English.xlf

```diff
@@ -7912,6 +7912,10 @@ To show all branches, right click the revision grid, select 'view' and then the
         <source>&amp;Manage...</source>
         <target />
       </trans-unit>
+      <trans-unit id="mnuBtnOpenRemoteUrlInBrowser.Text">
+        <source>Open remote url</source>
```

Comment:
I'm on the phone. Can you please confirm we use "url" in other places too? Or "URL", etc.

Reply:
Looking at the all file, that's mostly 'Url' even if there are some 'URL' (2 or 3) .

Indeed, that's the only 'url'. 

I will update that but should we standardize around 'Url'?